source/emitCustomAttributes.ts

```diff
@@ -56,12 +56,23 @@ export function emitCustomAttributes(
         .map(
           (schema) => `
               ${schema.id}: ${
-            customAttributes
-              .filter(
-                (x) => x.is_hierarchical || x.object_type?.name === schema.id
-              )
+            chain(customAttributes)
+              .filter((x) => {
+                const alias =
+                  typeof schema.alias_for === "string"
+                    ? schema.alias_for
+                    : schema.alias_for?.id;
+                return (
+                  x.is_hierarchical ||
+                  x.object_type?.name === schema.id ||
+                  (x.entity_type === "show" &&
+                    alias?.toLowerCase() === x.entity_type)
```

Comment:
The part I'm not getting, is this not simply `alias?.toLowerCase() === "show"`?

Reply:
If I do that, the project will have all attributes on it, and I think that's wrong. I just tested it, and then the snapshot test fails and puts all attributes on the project.

Reply:
In other words, if I only checkf for `alias` to be "show", then it will go through examples where `x.entity_type` is not also "show".

So actually the above expression is the same as:

```typescript
(x.entity_type === "show" && alias?.toLowerCase() === "show)
```

Reply:
Oh, of course. We're inside the map of `schemas` (That has the `alias`) and then we're filtering the `customAttributes`.